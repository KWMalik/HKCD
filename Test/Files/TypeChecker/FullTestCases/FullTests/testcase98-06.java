// Invalid test
class Test7a {

	public static void main(String[] args) {
		int x = 10;
		int y = 9;

		if(x > 0) {
			x = 0;
		}
		else {
			y = 0;
		}
	}
}

class Test7a {

}
