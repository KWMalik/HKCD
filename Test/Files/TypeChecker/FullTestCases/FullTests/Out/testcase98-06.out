Type error on line 14: Multiple declarations found for class Test7a.
Type error on line 18: Multiple declarations found for class Test7a.